### DISCUSSION TITLE
How to change a property's data type (a data type alias)

### DISCUSSION THREAD

**Discussion by youssefbennour** [ServiceStack/Discuss#80] [Q&A] [answered]
So, I have a property of type 'object' which on insert or update, I save as a string (for some good reason).
I want to ask about how does OrmLite interprets the 'object' data type, does it interpret it as string by default ?
If it ignores it, how can I make an alias-like data type for the property (i.e. make the POCO's property of type 'object' but have it's corresponding column of type  string/nvarchar in the database ?

**Comment by mythz** [comment, marked as answer]
You should avoid saving unknown types like `object` as serialization libraries won't know what Type to deserialize back into without embedding non-standard type info in the serialization wire format which isn't supported by most serializers.

I'd recommend either saving the concrete Complex Type which will be de/serialized by the RDBMS registered [Complex Type Serializer](https://docs.servicestack.net/ormlite/complex-type-serializers) or if you want to serialize unknown payloads serialize it as a `string` and have your App handle what unknown type to deserialize it back into.
